spring-cloud-gcp-examples/spring-cloud-gcp-sql-example/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.cloud.gcp.sql.database-name=example
+spring.cloud.gcp.sql.instance-connection-name=sodium-gateway-790:us-central1:test
+
+# So app starts despite "table already exists" errors.
+spring.datasource.continue-on-error=true
```

Comment:
Maybe also add `#spring.cloud.gcp.sql.credentials-location=file:/path/to/service-account.json` and `#spring.cloud.gcp.sql.password=secret` commented out for convenience.